canonical-web-and-design/snapcraft.io
# Get GitHub repos filtered by org

## Done

- Added route "/publisher/github/get-repos" that return a json
- List of GitHub Orgs passed to the template

## Issue / Card

Fixes #2486 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to /<snap_name>/builds and login with github
- Now you should be able to see an HTML select with all your GitHub organizations
- Check that http://0.0.0.0:8004/publisher/github/get-repos return a JSON with all your repos
- Check that http://0.0.0.0:8004/publisher/github/get-repos?org=canonical-web-and-design return a JSON with the Canonical repos.
